plugins/hydra_rq_launcher/README.md

```diff
@@ -0,0 +1,67 @@
+# Hydra RQ Launcher
+
+Provides a [Redis Queue (RQ)](https://python-rq.org) launcher for Hydra:
+
+> RQ (Redis Queue) is a simple Python library for queueing jobs and processing them in the background with workers.
+
+
+### Installation
+
+This plugin requires Hydra 1.0 (Release candidate)
+
+```commandline
+pip install -e .
+```
+
+Usage of this plugin requires a Redis server.
+
+
+### Usage
+
+Once installed, add `hydra/launcher=rq` to your command line. Alternatively, override `hydra/launcher` in your config:
+
+```yaml
+defaults:
+  - hydra/launcher: rq
+```
```

Comment:
You can show the output of `python foo.py --cfg hydra --package hydra.launcher` as documentation for what the plugin config looks like with one example for overriding from the command line.

Also, link to plugins/hydra_rq_launcher/hydra_plugins/hydra_rq_launcher/config.py in the repo for additional documentation of the types.